packages/validate/src/ValidateMixin.js

```diff
@@ -479,15 +480,19 @@ export const ValidateMixin = dedupeMixin(
         return JSON.stringify(result) !== JSON.stringify(prevResult);
       }
 
-      __isEmpty(v) {
+      __isEmpty() {
         if (typeof this._isEmpty === 'function') {
-          return this._isEmpty(v);
+          return this._isEmpty();
         }
         // // TODO: move to compat layer. Be sure to keep this, because people use this a lot
         // if (typeof this.__isRequired === 'function') {
         //   return !this.__isRequired(v);
         // }
-        return v === null || typeof v === 'undefined' || v === '';
+        return (
```

Comment:
This code is probably never reached because of the fact that we depend on FormControlMixin and that one defines _isEmpty?

Maybe we can get rid of `__isEmpty` and call `_isEmpty` directly inside `validate()`?

Reply:
Added a TODO for removing this by the time that we have a form package, so we can more easily rely on that (currently implicit) dependency